docs/resources/ddm_schema.md

```diff
@@ -0,0 +1,183 @@
+  security_group_id = huaweicloud_networking_secgroup.test.id
+
+  availability_zones = [
+    data.huaweicloud_availability_zones.test.names[0]
+  ]
+}
+
+resource "huaweicloud_rds_instance" "test" {
+  name              = "rds_test"
+  flavor            = "rds.mysql.n1.large.4"
+  security_group_id = huaweicloud_networking_secgroup.test.id
+  subnet_id         = huaweicloud_vpc_subnet.test.id
+  vpc_id            = huaweicloud_vpc.test.id
+  fixed_ip          = "192.168.0.58"
+
+  availability_zone = [
+    data.huaweicloud_availability_zones.test.names[0]
+  ]
+
+  db {
+    password = "test_password_123"
+    type     = "MySQL"
+    version  = "5.7"
+    port     = 3306
+  }
+
+  volume {
+    type = "CLOUDSSD"
+    size = 40
+  }
+}
+
+resource "huaweicloud_ddm_schema" "test"{
+  instance_id  = huaweicloud_ddm_instance.test.id
+  name         = "test_schema"
+  shard_mode   = "single"
+  shard_number = 1
+
+  data_nodes {
+    id             = huaweicloud_rds_instance.test.id
+    admin_user     = "root"
+    admin_password = "test_password_123"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of a DDM instance.
+
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the DDM schema.
+  An instance name starts with a letter, consists of 2 to 48 characters, and can contain only lowercase letters,
+  digits, and underscores (_). Cannot contain keywords information_schema, mysql, performance_schema, or sys.
+
+  Changing this parameter will create a new resource.
+
+* `shard_mode` - (Required, String, ForceNew) Specifies the sharding mode of the schema. Values option: **cluster**, **single**.
+  **cluster**: indicates that the schema is in sharded mode.
+  **single**: indicates that the schema is in unsharded mode.
+
+  Changing this parameter will create a new resource.
+
+* `shard_number` - (Required, Int, ForceNew) Specifies the number of shards in the same working mode.
+  If shard_unit is not empty, the value is the product of shard_unit multiplied by the associated RDS instances.
+  If shard_unit is left blank, the value must be greater than the number of associated RDS instances and
+  less than or equal to the product of the associated RDS instances multiplied by 64.
+
+  Changing this parameter will create a new resource.
+
+* `data_nodes` - (Required, List, ForceNew) Specifies the RDS instances associated with the schema.
+
+  Changing this parameter will create a new resource.
+  The [DataNode](#DdmSchema_DataNode) structure is documented below.
+
+* `shard_unit` - (Optional, Int, ForceNew) Specifies the number of shards per RDS instance.
```

Comment:
as discussed, this parameter is useless, please remove it

Reply:
done